test/common/CollectiveArgs.hpp

```diff
@@ -72,6 +74,22 @@ namespace RcclUnitTesting
     char ptr[MAX_RANKS * sizeof(double)];
   };
 
+  struct OptionalColArgs
+  {
+    int      collId = -1;
+    int      rank = -1;
+    PtrUnion scalarsPerRank = {nullptr};
+    int      scalarMode = -1;
+    // allToAllv args
+    size_t   sendcounts[MAX_RANKS*MAX_RANKS];
+    size_t   sdispls[MAX_RANKS*MAX_RANKS];
+    size_t   recvcounts[MAX_RANKS*MAX_RANKS];
+    size_t   rdispls[MAX_RANKS*MAX_RANKS];
+    size_t   numInputElementsArray[MAX_RANKS];
+    size_t   numOutputElementsArray[MAX_RANKS];
```

Comment:
Is there any reason to store this data?  It looks like these values are set but never actually used.